recipe/meta.yaml

```diff
@@ -12,8 +12,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  skip: true  # [win and py27 or win32]
+  number: 1
```

Comment:
Bump the `build/number` as re-rendering moves this recipe from CentOS 6 to 7 and bump the macOS `x86_64` minimum version from 10.9 to 10.13